packages/cactus-plugin-ledger-connector-aries/openapitools.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$schema": "node_modules/@openapitools/openapi-generator-cli/config.schema.json",
+  "spaces": 2,
+  "generator-cli": {
+    "version": "6.3.0"
```

Comment:
@outSH Could this be set to 6.6.0 to match the other packges?
There are race conditions with downloading the open api generator jar files and that's why we have a script that first download the 6.6.0 jar file and then the codegen scripts run, but if you deviate then the race condition will pop up again because the auto-download feature of the openapigenerator is the one with the race condition and without the pre-download it will default to the auto-download feature.

Reply:
Sure, done the same in ethereum and sawtooth connectors too

Reply:
@outSH Triple thank you!